peer/peer.go

```diff
@@ -2289,7 +2434,12 @@ func NewOutboundPeer(cfg *Config, addr string) (*Peer, error) {
 		}
 		p.na = na
 	} else {
-		p.na = wire.NewNetAddressIPPort(net.ParseIP(host), uint16(port), 0)
+		// If host is an onion hidden service, it is likely that a
```

Comment:
Can we prevent this at the API level? Otherwise, I think we should update the `godoc` to warn callers.

Reply:
It can be prevented with an error right there rather than continuing

Reply:
I updated the godoc, `wire.NetAddressV2FromBytes` would need to return an error to be able to fail gracefully here.